openshift/settings.sh

```diff
@@ -1,2 +1,3 @@
-export PROJECT_NAMESPACE=${PROJECT_NAMESPACE:-"bcgov"}
-export GIT_URI=${GIT_URI:-"https://github.com/BCDevOps/StorageMigration.git"}
\ No newline at end of file
+export PROJECT_NAMESPACE=${PROJECT_NAMESPACE:-"devex-von"}
+export GIT_URI=${GIT_URI:-"https://github.com/BCDevOps/StorageMigration.git"}
+export GIT_REF=${GIT_REF:-"master"}
```

Comment:
I am thinking this file is a leftover from before having the `manage` script and it should go. @WadeBarnes can you please confirm?

Reply:
Required, even for the `manage` script(s).